include/sys/mem_manage.h

```diff
@@ -23,6 +23,14 @@
 /** Full write-back caching. Any RAM mapped wants this. */
 #define K_MEM_CACHE_WB		0
 
+/*
+ * Include ARM64 Specific flags, pay attention to be not conflicted
+ * when updating these flags.
+ */
+#if defined(CONFIG_ARM_MMU) || defined(CONFIG_ARM_MPU)
+#include <arch/arm64/mm.h>
+#endif
+
```

Comment:
why here? please move this at the correct location for the header files

Reply:
I thought it should follow the public definition, but anyway I moved it to be beginning of the header file, how about now? thanks.